### DISCUSSION TITLE
Login Error

### DISCUSSION THREAD

**Discussion by mhouser78** [AdguardTeam/AdGuardHome#5953] [Q&A]
I get this when I log into my Adguard home docker container Error:
control/login | invalid username or password | 403

I tried to edit the AdGuardHome.yaml to no avail. I did restart the
Docker container after changing the password.

**Comment by Freebase394** [comment]
Simply reinstall your AGH or Folow this topic 
https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#password-reset

Please, always use the internet and google at your disposal!  Believe me, it will always be much faster!
